.github/workflows/openapi_validation.yml

```diff
@@ -0,0 +1,49 @@
+name: Validate OpenAPI Schema
+
+on:
+  pull_request:
+    branches:
+      - master
```

Comment:
Can we have this for the draft branch too? Generally all changes are pushed to draft before they are moved to master.

Reply:
Sure, that can be done. Will get it done, by tomorrow EOD.